x-pack/platform/plugins/shared/security/server/authentication/authenticator.ts

```diff
@@ -758,10 +761,12 @@ export class Authenticator {
     request: KibanaRequest,
     {
       provider,
+      providerInstance,
```

Comment:
__optional nit:__ it's missing in the JSDocs (not a big deal though).

Reply:
Will fix in subsequent PR